ur_description/env-hooks/ur_description.dsv.in

```diff
@@ -0,0 +1 @@
+prepend-non-duplicate;IGN_GAZEBO_RESOURCE_PATH;share
```

Comment:
Please add a comment at the top of this file. What is its purpose? Why did you put it in the ur_description folder?

Reply:
Oh this is an environment hook that add an environment variable called `IGN_GAZEBO_RESOURCE_PATH` when this package is sourced, so that Gazebo can find the meshes.

Reply:
See https://github.com/ros-simulation/gazebo_ros_pkgs/wiki/ROS-2-Migration:-Gazebo-ROS-Paths

Reply:
Can you still add a comment for others in the future, please

Reply:
I am not sure about .dsv syntax but wasn't happy with commenting. I did add a comment about this in CMakeLists.txt though in the last commit.